lib/common/grpc-service.js

```diff
@@ -266,6 +266,25 @@ GrpcService.prototype.request = function(protoOpts, reqOpts, callback) {
     request: function(_, onResponse) {
       respError = null;
 
+      if (stream) {
```

Comment:
That worries me a little bit, only because I think BigTable is often used for extremely large datasets. Not to mention that if we did want to offer this in the form of a stream (without touching stream-router) then we would have to transform a stream to a callback and then back to a stream, which feels pretty inefficient.